readme.md

```diff
@@ -435,17 +434,11 @@ class Hello extends Component {
 Ok, we set an initial state. But how do we go about changing it?
 * We need to set up some sort of trigger event to change our counter.
 
-<details>
-  <summary><strong>
-    Let's do that via a button click event. Where should we initialize it?
-  </strong></summary>
-
-  > In the return value of our Post's `render` method.
-
-</details>
+Take a look at how this event is implemented. We use an attribute called `onClick` to define the behavior as to what happens when we click this particular button. As it's value, we're passing in an anonymous function that invokes handleClick, a function defined on this component.
```

Comment:
- `define the behavior as to what happens` --> `define what happens`
- `it's` --> `its`